openapi/paths/invoices@{id}@credit-memo-allocations.yaml

```
@@ -0,0 +1,36 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Invoices
+  summary: Get credit memo's amounts allocated to an invoice.
```

```suggestion
  summary: Get credit memo allocations to invoices
```

Wouldn't it be one allocation per invoice?